MinecraftClient/Mapping/Dimension.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MinecraftClient.Mapping
+{
+
+    /// <summary>
+    /// The dimension type, available after 1.16.2
+    /// </summary>
+    public class Dimension
```

Comment:
Should dimension fields have default values in case they are not set by the server?